app/scripts/modules/kubernetes/loadBalancer/configure/wizard/upsert.controller.js

```diff
@@ -148,11 +148,11 @@ module.exports = angular.module('spinnaker.loadBalancer.kubernetes.create.contro
     this.submit = function () {
       var descriptor = isNew ? 'Create' : 'Update';
 
+      this.updateName();
```

Comment:
The fact that I had `loadBalancerName` came from quickly reading the GCE code, and my failure to read what the `loadBalancerWriter.upsertLoadBalancer` does with `params`. Turns out it just includes it in the request to orca/clouddriver. My clouddriver code reads the `name` field to determine the name of the load balancer, so `loadBalancerName` is extraneous. The last call to `updateName` makes sure the current `$scope.loadBalancer` has the correct name set. I ran into a bug where it wasn't always current, prompting this change.